source/installation/install-kafka-cluster.txt

```diff
@@ -0,0 +1,146 @@
+.. _install-kafka-cluster:
+
+======================================
+Install with an Existing Kafka Cluster
+======================================
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can install Relational Migrator on an existing Kafka cluster including 
+Kafka Connect. This installation method is ideal if you are already running 
+Kafka or you have your own approach to installing and configuring Kafka.
+
+Before you Begin 
+----------------
+
+You must have Kafka and a Kafka Connect cluster accessible to the computer 
+running Relational Migrator.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
+   .. step:: Install Debezium Source Connect Plugin
+
+      1. Download the source connector archive for your source database.
+
+         - `MySQL Connector plugin archive 
+           <https://repo1.maven.org/maven2/io/debezium/debezium-connector-mysql/2.4.1.Final/debezium-connector-mysql-2.4.1.Final-plugin.tar.gz>`__
+         - `Postgres Connector plugin archive 
+           <https://repo1.maven.org/maven2/io/debezium/debezium-connector-postgres/2.4.1.Final/debezium-connector-postgres-2.4.1.Final-plugin.tar.gz>`__
+         - `SQL Server Connector plugin archive 
+           <https://repo1.maven.org/maven2/io/debezium/debezium-connector-sqlserver/2.4.1.Final/debezium-connector-sqlserver-2.4.1.Final-plugin.tar.gz>`__ 
+         - `Oracle Connector plugin archive 
+           <https://repo1.maven.org/maven2/io/debezium/debezium-connector-oracle/2.4.1.Final/debezium-connector-oracle-2.4.1.Final-plugin.tar.gz>`__  
+         - `Db2 Connector plugin archive 
+           <https://repo1.maven.org/maven2/io/debezium/debezium-connector-db2/2.4.1.Final/debezium-connector-db2-2.4.1.Final-plugin.tar.gz>`__
+
+      #. Unzip the archive.
+
+      #. Copy the ``debezium-connector.jar`` file to the ``debezium-connector`` directory.
+
+         From the downloaded archive, copy the ``debezium-connector.jar`` file to the
+         ``debezium-connector-xxx`` directory at Kafka Connect’s `plugin.path 
+         <https://kafka.apache.org/documentation/#connectconfigs_plugin.path>`__ on your 
+         Kafka Connect cluster. For example: ``/kafka/connect/debezium-connector-sqlserver``.
+
+      #. Restart you Kafka Connect cluster.
+       
+         To know if your plugin is successfully registered, you must see the following 
+         log statement in the ``kafka-connect`` start up logs:
+
+         .. code-block::
+            :copyable: false
+
+            INFO || Added plugin 'io.debezium.connector.sqlserver.SqlServerConnector'
+
+   .. step:: Install Relational Migrator Sink Connect Plugin
+
+      1. Download the latest version of the ``kafka-connect.jar`` file.
```

Comment:
Should this be ``kafka-connect-migrator.jar``?

```suggestion
      1. Download the latest version of the ``kafka-connect-migrator.jar`` file.
```
![Screenshot 2024-03-15 at 12 20 51 PM](https://github.com/mongodb/docs-relational-migrator/assets/85948430/d2080f8f-c00a-465e-8202-32e4b53d00ab)

Reply:
Updated.